prime-router/src/main/kotlin/fhirengine/engine/FHIRRouter.kt

```diff
@@ -243,7 +245,12 @@ class FHIRRouter(
             //           must have at least one of patient street, zip code, phone number, email
             //           must have at least one of order test date, specimen collection date/time, test result date
             passes = passes &&
-                evaluateFilterCondition(getQualityFilters(receiver, orgFilters), bundle, qualFilterDefaultResult)
+                evaluateFilterCondition(
+                    getQualityFilters(receiver, orgFilters),
+                    bundle,
+                    qualFilterDefaultResult,
+                    receiver.reverseTheQualityFilter
```

Comment:
You also need a unit test to test that the receiver's config is used correctly here.